src/Compilers/CSharp/Portable/Parser/CharacterInfo.cs

```diff
@@ -345,28 +275,12 @@ internal static bool IsFormattingChar(char ch)
         /// <param name="cat">The Unicode character.</param>
         internal static bool IsFormattingChar(UnicodeCategory cat)
         {
-            // formatting-character:  
-            //   A Unicode character of the class Cf
-            //   A unicode-escape-sequence representing a character of the class Cf
-
-            return cat == UnicodeCategory.Format;
+            return UnicodeCharacterUtilities.IsFormattingChar(cat);
         }
 
         internal static bool IsNonAsciiQuotationMark(char ch)
```

Comment:
No, but it wasn't specific to C# syntax facts. I'll restore it back as only the C# compiler needs it.